### ISSUE TITLE
Updating a nuget, updates file reference

### ISSUE THREAD

**Issue by haf** [fsprojects/Paket#2626]
### Description

Run `paket update dotliquid` in suave's code base. It will update YoLo.fs as well.

### Repro steps

1. Download the Suave code base (from the time of writing)
2. Run `paket update DotLiquid` 
3. See the diff include "YoLo.fs" despite the update being scoped to a single package.

### Expected behavior

Not to update other things than specified.

### Actual behavior

IRL see problems faced by contributor here https://github.com/SuaveIO/suave/issues/641#issuecomment-323151091

### Known workarounds

Hack around in VIM after updating the package, to fix it. Or just update the package manually by first going to nuget.org, checking the version, and then inputting that in paket.lock, whereafter running `paket install` is necessary.

**Comment by forki** [comment]
Yes this is a long standing known issue. Thanks for nudge. Hopefully I find time for it soon.

**Comment by haf** [comment]
Closing as duplicate then.

**Comment by matthid** [comment]
Where is the duplicate?

**Comment by haf** [comment]
I just thought so since forki wrote "Yes this is a long standing known issue"...

Let's reopen it then!

**Comment by forki** [comment]
not sure if dupe, but it's in my head as one of the ackwards things that I always postpone

**Comment by forki** [comment]
good to have it filed properly

**Comment by haf** [comment]
Ping

**Comment by forki** [comment]
no update on this. sorry

**Comment by haf** [comment]
ok, thanks for letting me know!
